src/core/Track.cpp

```diff
@@ -735,8 +735,11 @@ void TrackContentObjectView::paintTextLabel(QString const & text, QPainter & pai
  */
 void TrackContentObjectView::mousePressEvent( QMouseEvent * me )
 {
-	QVector<TrackContentObjectView *> active = getClickedTCOs();
-	// active will be later used for the remove, copy, cut or toggleMute methods
+	// Right now, active is only used on right/mid clicks actions, so we use a ternary operator
+	// to avoid the overhead of calling getClickedTCOs when it's not used
+	auto active = me->button() == Qt::LeftButton
+							? QVector<TrackContentObjectView *>()
+							: getClickedTCOs();
```

Comment:
I think two tabs over ternary options would be best, not 7 tabs.

Reply:
It looks like you're using tabs for alignment, which isn't good. If someone displays tabs as 2 spaces, these lines wouldn't start to the right of the above. If they use 8 spaces, they would be *extremely* far out.

If you want to align the cases with the end of the row above, you'd want to tab until the start of the lines line up, and then use spaces from there. This preserves alignment between lines at all tab sizes.

However, I agree with Veratil that we could stick to just indenting one extra tab.

Reply:
My bad, even to the alignment I wanted I was off. I thought about aligning it with the conditional but I went further. I kept it as an extra tab as suggested.